objects/spread.go

```diff
@@ -0,0 +1,13 @@
+package objects
+
+// Spread represents a value that will been spread onto the stack by the VM
+type Spread struct {
+	ObjectImpl
+	Values []Object
+}
+
+// TypeName returns the name of the type.
+func (s *Spread) TypeName() string { return "spread-values" }
+
+// String returns a string representation of the type's value.
+func (s *Spread) String() string { return "<" + s.TypeName() + ">" }
```

Comment:
Can this just be `return "<spread-values>"`?

Reply:
yep, fixed